composer.json

```diff
@@ -83,7 +77,8 @@
     "minimum-stability": "dev",
     "config": {
         "allow-plugins": {
-            "kylekatarnls/update-helper": false
+            "kylekatarnls/update-helper": false,
+            "php-http/discovery": false
```

Comment:
Installing the SDK on its own does not prompt the execution of `php-http/discovery`.
Why should we add this?

Reply:
This setting does not affect users of the package, it is only relevant for contributors. When I ran `composer install`, it prompted me to either allow or deny this. I did not deem it necessary, so I denied.